generators/app/templates/webapp/index.html

```diff
@@ -4,29 +4,29 @@
 		<meta http-equiv="Cache-control" content="no-cache, no-store, must-revalidate" />
 		<meta http-equiv="Pragma" content="no-cache" />
 		<meta http-equiv="expires" content="0" />
+
 		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Why? The App control in https://github.com/ui5-community/generator-ui5-ts-app/blob/main/generators/app/templates/webapp/view/App.view.xml#L7 does the same thing, I think.

Reply:
@akudev I really believe this was added because it also previously existed in the `index-cdn.html` file. But I think [you're right](https://github.com/SAP/openui5/blob/55d2a0b37c8e928786975008077cd815d43a2a26/src/sap.m/test/sap/m/qunit/App.qunit.js#L52-L61), so it may be removed from both? :)